exercises/practice/difference-of-squares/.meta/solutions/DifferenceOfSquares.pm

```diff
@@ -5,34 +5,37 @@ use Exporter qw<import>;
 our @EXPORT_OK
   = qw<square_of_sum sum_of_squares difference_of_squares>;
 
+use lib 'lib';
+use Exercism::QuickSolve;
+
 sub square_of_sum {
   my ($number) = @_;
-  return (
-    grep {
-           $_->{property} eq 'squareOfSum'
-        && $_->{input}{number} eq $number
-    } @main::test_cases
-  )[0]->{expected};
+
+  quicksolve(
+    input     => $number,
+    input_key => 'number',
+    property  => 'squareOfSum',
```

Comment:
These alignments, they look good. There are other places where this style could be adhered to.  Keeping a consistency is  a good thing.  I proposed a change in a different comment.